### DISCUSSION TITLE
有办法在列头文字前面加上自定义icon么，不需要有action的能力，纯粹是展示

### DISCUSSION THREAD

**Discussion by Chasen-Zhang** [antvis/S2#1845] [Q&A]
![image](https://user-images.githubusercontent.com/15354455/197384127-60be4113-ee4e-4fd4-81ee-358aaf6f5ccc.png)
有办法在列头文字前面加上自定义icon么，不需要有action的能力，纯粹是展示

**Comment by lijinke666** [comment]
后面可以考虑 `headerActionIcons` 提供 `position` 的配置

目前可以自定义列头, 把文字的坐标往前移, 预留 一个 icon 的位置, 然后通过 `this.addActionIcon()` 绘制 icon

需要查看下源码, 参考内置 icon 绘制的方式

https://github.com/antvis/S2/blob/80cbf2060ac530e0edeae1f7adef7a56303f8723/packages/s2-core/src/cell/header-cell.ts#L232

https://github.com/antvis/S2/blob/80cbf2060ac530e0edeae1f7adef7a56303f8723/packages/s2-core/src/cell/header-cell.ts#L183

**Comment by Chasen-Zhang** [comment]
我发现renderIcon 这函数可以满足我的需求。感谢

**Comment by lijinke666** [comment]
6
